sdk/operationalinsights/Microsoft.Azure.OperationalInsights/src/Microsoft.Azure.OperationalInsights.csproj

```diff
@@ -2,9 +2,9 @@
   <PropertyGroup>
     <Description>Provides Microsoft Azure Operational Insights data operations</Description>
     <AssemblyTitle>Microsoft Azure Operational Insights Data Library</AssemblyTitle>
-    <Version>1.0.0-preview.1</Version>
+    <Version>0.10.1</Version>
```

Comment:
Why is this moving to version .10 instead of version 1?

Reply:
Yes, thanks!